articles/custom-domains/set-up-azure-cdn.md

```diff
@@ -0,0 +1,144 @@
+| Pricing tier | Select **Premium Verizon**. |
+
+![](/media/articles/custom-domains/azure/cdn-profile-1.png)
+
+Underneath the parameters mentioned above, you'll see an option to **create a new CDN endpoint now**. Make sure that you check the box next to this option. You'll need to provide a few additional configuration values:
+
+| Parameter | Value |
+| - | - |
+| CDN endpoint name | A descriptive name for the endpoint. We suggest using something similar to your custom domain (if not the domain itself) |
+| Origin type | Select **Custom origin** |
+| Origin hostname | Provide the fully-qualified URL using the following format: `www.tenant.custom_domain_id.edge.tenants.auth0.com`. Be sure to replace the `tenant` (this is the name of your Auth0 account and is reflected in your domain name as `tenant.auth0.com`) and `custom_domain_id` placeholder values. 
+
+::: warning
+If your Auth0 tenants are **not** in a US region, you'll need to use the following as your **Origin hostname**:
+
+* EU: tenant.custom_domain_id.edge.tenants.eu.auth0.com
+* AU: tenant.custom_domain_id.edge.tenants.au.auth0.com
+
+Be sure to replace the `tenant` placeholder value with the name of your tenant.
+:::
+
+![](/media/articles/custom-domains/azure/cdn-profile-2.png)
+
+Click **Create** to proceed.
+
+While Azure creates your CDN, it'll redirect you to the Azure Portal Dashboard. You can check on its progress using the drop-down menu under the alarm bell icon.
+
+![](/media/articles/custom-domains/azure/creating-profile.png)
+
+When done, you'll see your new CDN profile and Endpoint under your list of resources.
+
+![](/media/articles/custom-domains/azure/resources.png)
+
+### Finish Configuring the Endpoint
+
+At this point, we will need to finish configuring your newly-created endpoint. Under your list of resources (you can find this on the Dashboard page), select your endpoint to launch its configuration area.
+
+![](/media/articles/custom-domains/azure/endpoint.png)
+
+In the near-left column (under **Settings**), click **Origin**.
+
+![](/media/articles/custom-domains/azure/endpoint-origin.png)
+
+Check that the **Origin hostname** and the **Origin host header** values are identical. Then, uncheck the **HTTP** box to disable the protocol (you want to allow only calls using the HTTPS protocol).
+
+![](/media/articles/custom-domains/azure/disable-http.png)
+
+Return to the top of the window and click **Save**.
+
+### Finish Configuring the CDN Profile
+
+You will need to [add your custom domain name to your Azure CDN endpoint](https://docs.microsoft.com/en-us/azure/cdn/cdn-map-content-to-custom-domain#register-a-custom-domain-for-an-azure-cdn-endpoint-using-the-intermediary-cdnverify-subdomain).
+
+However, before you proceed, create a CNAME entry in your DNS record for the custom domain name that points to the CDN endpoint name. It'll look something like this: `login.mydomain.com CNAME login-mydomain-com.azureedge.net`.
```

Comment:
Creating a cname record from your custom domain to the Azure CDN is something we also need to include in the AWS CloudFront instructions (I added a separate note).